app/models/agents/website_agent.rb

```diff
@@ -379,21 +382,24 @@ def handle_data(body, url, existing_payload)
             extract_xml(doc)
         end
 
-      if output.each_value.each_cons(2).any? { |m, n| m.size != n.size }
-        raise "Got an uneven number of matches for #{interpolated['name']}: #{interpolated['extract'].inspect}"
-      end
-
-      num_tuples = output.each_value.first.size
+      num_tuples = output.each_value.inject(nil) { |num, value|
+        case size = value.size
+        when Float::INFINITY
```

Comment:
This is what Repeater#size returns, meaning it is an infinite list.  `num_tuples` is being set to a finite number that is the size of finite lists, so it is counted out here.